Source/Scene/ModelExperimental/ModelExperimentalSceneGraph.js

```diff
@@ -347,9 +348,18 @@ ModelExperimentalSceneGraph.prototype.buildDrawCommands = function (
       runtimePrimitive.drawCommands = drawCommands;
     }
   }
-  this._boundingSphere = BoundingSphere.fromBoundingSpheres(
+
+  const scaledBoundingSphere = BoundingSphere.fromBoundingSpheres(
     this._boundingSpheres
   );
+  scaledBoundingSphere.center = Cartesian3.multiplyByScalar(
+    scaledBoundingSphere.center,
+    this._model.scale,
+    scaledBoundingSphere.center
+  );
+  scaledBoundingSphere.radius = this._model.scale * scaledBoundingSphere.radius;
+
+  this._boundingSphere = scaledBoundingSphere;
```

Comment:
I need to review this section is more detail later, but first could you merge `main` into this branch? The code changed a bit in https://github.com/CesiumGS/cesium/pull/10078.